index.js

```diff
@@ -31,17 +31,18 @@ app.get('*', (req, res) => {
     req.query.lang =  req.query.lang.toLowerCase()
   }
 
-
   if (redirectRequired) {
-      params = req.query;
-      params = Object.keys(params).map(key => key + '=' + params[key]).join('&');
+    params = req.query;
+    params = Object.keys(params).map(key => key + '=' + params[key]).join('&');
     res.redirect(req.path + '?' + params)
+    return
```

Comment:
wonder if this one is necessary
Also, please use a linter, `;` is required at the end of every line.

Reply:
There's also mixed indentation across the file.
Some lines are indented with 2 spaces and some are with 4.

Reply:
About return: I read somewhere that if the function is not left (with return) the execution is actually continues.
I check the others.